pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -630,6 +630,16 @@ func (p *Provisioner) setAdminOpsAPIClient() error {
 		return errors.Wrapf(err, "failed to get ceph object store %q", p.objectStoreName)
 	}
 
+	cephCluster, err := p.getCephCluster()
+	if err != nil {
+		return errors.Wrapf(err, "failed to get ceph cluster in namespace %q", p.clusterInfo.Namespace)
+	}
+	if cephCluster == nil {
```

Comment:
Why would the cluster be nil? If `getCephCluster()` returns `Items[0]`, seems like it shouldn't be nil